tests/bsim/bluetooth/host/att/read_fill_buf/CMakeLists.txt

```diff
@@ -3,15 +3,15 @@
 cmake_minimum_required(VERSION 3.20.0)
```

Comment:
At this moment – yes. This is workaround for Twister which requires to have this “main app”. But I believe that this could be changed on a Twister side, or alternatively in Sysbuild (to give possibility of choosing which app is a “main app” by adding in `sysbuild.cmake` statement like `set(DEFAULT_IMAGE "client"...)`).